shared/templates/sysctl/oval.template

```diff
@@ -213,6 +221,13 @@
   <ind:textfilecontent54_object id="object_static_run_usr_local_sysctls_{{{ rule_id }}}" version="1">
     <set>
       <object_reference>object_static_usr_local_lib_sysctld_{{{ rule_id }}}</object_reference>
+      <object_reference>object_static_lib_sysctls_{{{ rule_id }}}</object_reference>
+    </set>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_object id="object_static_lib_sysctls_{{{ rule_id }}}" version="1">
+    <set>
+      <object_reference>object_static_lib_sysctld_{{{ rule_id }}}</object_reference>
```

Comment:
The reference to `object_static_lib_sysctld_{{{ rule_id }}}` shouldn't be included in RHEL datastreams.
I suggest you put in the same as set as `object_static_usr_lib_sysctld_{{{ rule_id }}}`, and then any reference to the set object should be guarded by the same conditional.

See suggestion below:
```
  <ind:textfilecontent54_object id="object_static_lib_sysctls_{{{ rule_id }}}" version="1">
    <set>
      <object_reference>object_static_run_sysctld_{{{ rule_id }}}</object_reference>
{{% if product not in [ "ol7", "ol8", "ol9", "rhcos4", "rhel7", "rhel8", "rhel9"] %}}
      <object_reference>object_static_lib_usr_lib_sysctls_{{{ rule_id }}}</object_reference>
{{% endif %}}
    </set>
  </ind:textfilecontent54_object>

{{% if product not in [ "ol7", "ol8", "ol9", "rhcos4", "rhel7", "rhel8", "rhel9"] %}}
  <ind:textfilecontent54_object id="object_static_lib_usr_lib_sysctls_{{{ rule_id }}}" version="1">
    <set>
      <object_reference>object_static_lib_sysctld_{{{ rule_id }}}</object_reference>
      <object_reference>object_static_usr_lib_sysctld_{{{ rule_id }}}</object_reference>
    </set>
  </ind:textfilecontent54_object>
{{% endif %}}
```

Reply:
Should be solved in [a6600ee](https://github.com/ComplianceAsCode/content/pull/10555/commits/a6600ee176c5f5feccc531f539594faa74719c6f)